osu.Game.Tests/Visual/Multiplayer/TestSceneRankRangePill.cs

```diff
@@ -73,22 +74,29 @@ public void TestMultipleUsers()
         [TestCase(1000000, 10000000)]
         public void TestRange(int min, int max)
         {
-            AddStep("add users", () =>
-            {
-                MultiplayerClient.AddUser(new APIUser
+            setupRoomWithUsers(
+                new APIUser
                 {
                     Id = 2,
                     Statistics = { GlobalRank = min }
-                });
-
-                MultiplayerClient.AddUser(new APIUser
+                },
+                new APIUser
                 {
                     Id = 3,
                     Statistics = { GlobalRank = max }
                 });
+        }
+
+        private void setupRoomWithUsers(params APIUser[] users)
+        {
+            AddStep("setup room", () =>
+            {
+                multiplayerClient.SetupGet(m => m.Room).Returns(new MultiplayerRoom(0)
+                {
+                    Users = new List<MultiplayerRoomUser>(users.Select(apiUser => new MultiplayerRoomUser(apiUser.Id) { User = apiUser }))
+                });
 
-                // Remove the local user so only the ones above are displayed.
-                MultiplayerClient.RemoveUser(API.LocalUser.Value);
+                multiplayerClient.Raise(m => m.RoomUpdated -= null);
```

Comment:
Just to confirm, is this `-= null` thing only a syntax hack on the moq side to be able to reference the event in an expression? I had a bit of a brain explosion when first reading this.

Reply:
Yep, it's in the xmldoc for `Raise`.